.github/config/rubocop_linter_action.yml

```diff
@@ -0,0 +1,67 @@
+# .github/config/rubocop_linter_action.yml
+
+# Description: The name of the check that will be created.
+# Valid Options: A reasonably sized string.
+# Default: 'RuboCop Action'
+check_name: "RuboCop Results"
+
+# Description: Versions required to run your RuboCop checks.
+# Valid options: RuboCop and any RuboCop extension, by default the latest gem version will be used. You can explicitly state that
+# (not required) or use a version number, like '1.5.1'.
+# Default:
+#   versions:
+#     - rubocop: 'latest'
+versions:
+  - rubocop
+  - rubocop-rails
+  - rubocop-minitest
+  - rubocop-performance: "1.5.1"
+  - rubocop-rspec: "1.37.0"
+  - rubocop-rootstrap
+
+# Description: RuboCop configuration file path relative to the workspace.
+# Valid options: A valid file path inside of the workspace.
+# Default: nil
+# Note: This does not need to be filled out for RuboCop to still find your config.
+# Resource: https://rubocop.readthedocs.io/en/stable/configuration/
+rubocop_config_path: ".rubocop.yml"
+
+# Run all cops enabled by configuration except this list.
+# Valid options: list of valid cop(s) and/or departments.
+# Default: nil
+# Resource: https://rubocop.readthedocs.io/en/stable/cops/
+rubocop_excluded_cops:
+  - "Style/FrozenStringLiteralComment"
```

Comment:
Why are you excluding this rule?

Reply:
that is default excluding for the library

Reply:
the exclude was removed